src/model/transaction/splitBlockInContentState.js

```diff
@@ -46,6 +46,7 @@ function splitBlockInContentState(
     key: keyBelow,
     text: text.slice(offset),
     characterList: chars.slice(offset),
+    data: Object(),
```

Comment:
Should the lower block never keep the same `data`? It might depend on how the `data` field is being used. Thoughts?